radixdlt-java-common/src/main/java/com/radixdlt/identifiers/Rri.java

```diff
@@ -103,43 +102,14 @@ public static Rri fromBech32(String s) {
 		var d = Bech32.decode(s);
```

Comment:
`decode` can fail if not checksummed. Right now we just passthrough `org.bitcoinj.core.AddressFormatException`, I think it more produent to wrap this, and throw our own errors, saying that it was not checksummed (and all other errors)...